modules/combined/test/tests/invOpt_nonlinear/tests

```diff
@@ -0,0 +1,29 @@
+[Tests]
+  issues = '#22208'
+  design = 'LevelSetAdvection.md'
```

Comment:
not the right design doc. Use an optimization one

this is just a random advection kernel, that just happens to match what a 'CoupledAdvection' would look like